src/_scss/pages/award/awardFinancialAssistance.scss

```diff
@@ -23,6 +23,66 @@
                     padding: 0;
                     margin: 0;
                 }
+
```

Comment:
@jameslee40 looking good. Maybe we can bring up the orange color of pulse animation in design review? And move them further right with some margin?

Reply:
<img width="785" alt="Screen Shot 2020-09-17 at 1 31 50 PM" src="https://user-images.githubusercontent.com/14301977/93506563-752adb00-f8ea-11ea-8abc-5834910c98a3.png">

Reply:
updated color of pulse animation and added some margin!